aiogram/fsm/storage/base.py

```diff
@@ -19,6 +19,68 @@ class StorageKey:
     destiny: str = DEFAULT_DESTINY
 
 
+class KeyBuilder(ABC):
+    """Base class for key builder."""
+
+    @abstractmethod
+    def build(self, key: StorageKey, part: Literal["data", "state", "lock"]) -> str:
+        """
+        This method should be implemented in subclasses
+
+        :param key: contextual key
+        :param part: part of the record
+        :return: key to be used in storage's db queries
+        """
+        pass
```

Comment:
Redundant `pass`. This string may be removed

Reply:
`pass` is used within `@abstractmethod` in current codebase in many places. Therefore, it's better to leave it, so that everything is uniform.
If it should be removed from every module in codebase, seems like it's needed separate Pull Request with refactoring.